sdk/python/feast/feature_store.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         self._registry = Registry(
             registry_path=registry_config.path,
             repo_path=self.repo_path,
+            publish_json=registry_config.publish_json,
```

Comment:
Sorry for giving you so much back and forth, but I actually think `export_json` might be more descriptive than publish. What do you think? For the config option, argument, as well as the file name (`registry_export.json`)?

I just want to make sure its clear that this is a read only file and not the real registry, and I think export is more descriptive than publish.

Reply:
No worries, I think `export_json` and `registry_export.json` are much better names too. Thanks for the idea!